### PR TITLE
Feature/a11y home sections semantics

### PR DESCRIPTION
**Pull request by camilokawerin**
**What's broken**

Every section in a page must be marked with proper sectioning tag and must be labelled by a heading or `aria-label` attribute. This allows user with assistive technologies to navigate the page searching for landmarks in it.

**What kind of change does this PR introduce?**

Added sectioning tags and attributes where it is necessary.

### CODE REVIEW

File: src/components/LobbyHeroComponent.js [hunk 1]
```diff
@@ -4,23 +4,23 @@ import {connect} from "react-redux";
...
+  <div className="hero">
     <div className={`${styles.heroColumns} columns`}>
       <div className={`${styles.leftColumn} column is-black`}>
         <div className={`${styles.heroContainer} hero-body`}>
           <div className="container">
             <h1 className="title">
               {homeSettings.homeHero.title}
             </h1>
-            <h2 className="subtitle">
+            <p className="subtitle" role="doc-subtitle">
```
Comment (mrazzari) [inline src/components/LobbyHeroComponent.js:15]: Skip `role="doc-subtitle"`. It adds unnecesary levels to heading navigation. [Always trust Adrian](https://adrianroselli.com/2020/08/be-wary-of-doc-subtitle.html#Verdict).

**Review by mrazzari — CHANGES_REQUESTED** [review]
Minor change requested.

File: src/components/LobbyHeroComponent.js [hunk 2]
```diff
@@ -4,23 +4,23 @@ import {connect} from "react-redux";
 import styles from '../styles/lobby-hero.module.scss'
 
 const LobbyHeroComponent = ({homeSettings}) => (
-  <section className="hero">
+  <header className="hero">
```
Comment (mrazzari) [inline src/components/LobbyHeroComponent.js:7]: Let's go back to section.  (Avoid nesting header within main. Doesn't create a landmark, and if it did, wouldn't be valuable for users.)

**Review by mrazzari — CHANGES_REQUESTED** [review]
Minor change requested.

File: src/components/LobbyHeroComponent.js (hunk 2 above)
Comment (camilokawerin) [reply src/components/LobbyHeroComponent.js:7]: Then let's go to DIV, so it doesn't create a landmark at all. In fact, this is just a visual container for H1.

**Review by mrazzari — COMMENTED** [review]
LGTM

**Review by mrazzari — APPROVED** [review]

**Comment by smarcet** [comment]
@camilokawerin could you please fix the merge conflicts ?
much thanks!

**Comment by camilokawerin** [comment]
@smarcet done!

**Review by gcutrini — APPROVED** [review]

**Pull request merged**